### PR TITLE
Generate TFLite files from Tensorflow models

### PR DESCRIPTION
**Pull request by mariecwhite**

### CODE REVIEW

File: common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py [hunk 1]
```diff
@@ -61,6 +62,47 @@ def _generate_mlir(model_dir: pathlib.Path, saved_model_dir: pathlib.Path):
...
+  converter = tf.lite.TFLiteConverter.from_saved_model(str(saved_model_dir))
+
+  # Generate fp32 model.
+  try:
+    tflite_model = converter.convert()
+    tflite_model_path = model_dir.joinpath("model_fp32.tflite")
+    with open(tflite_model_path, 'wb') as f:
+      f.write(tflite_model)
+  except Exception as e:
+    print(f"Failed to generate int8 TFLite model. Exception: {e}")
```
Comment (pzread) [inline common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py:76]: typo: int8 -> fp32?

File: common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py [hunk 2]
```diff
@@ -61,6 +62,47 @@ def _generate_mlir(model_dir: pathlib.Path, saved_model_dir: pathlib.Path):
...
+def _generate_tflite(inputs: Tuple[Any, ...], model_dir: pathlib.Path,
+                     saved_model_dir: pathlib.Path):
+  converter = tf.lite.TFLiteConverter.from_saved_model(str(saved_model_dir))
+
+  # Generate fp32 model.
+  try:
+    tflite_model = converter.convert()
+    tflite_model_path = model_dir.joinpath("model_fp32.tflite")
+    with open(tflite_model_path, 'wb') as f:
+      f.write(tflite_model)
```
Comment (pzread) [inline common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py:74]: nit:
```suggestion
    tflite_model_path = model_dir / "model_fp32.tflite"
    tflite_model.path.write_bytes(tflite_model)
```

File: common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py [hunk 3]
```diff
@@ -61,6 +62,47 @@ def _generate_mlir(model_dir: pathlib.Path, saved_model_dir: pathlib.Path):
...
+    converter.optimizations = [tf.lite.Optimize.DEFAULT]
+    converter.target_spec.supported_ops = [
+        tf.lite.OpsSet.TFLITE_BUILTINS, tf.lite.OpsSet.TFLITE_BUILTINS_INT8
+    ]
+    converter.representative_dataset = representative_examples
+    converter.inference_type = tf.int8
+    tflite_model_int8 = converter.convert()
+    tflite_model_int8_path = model_dir.joinpath("model_int8.tflite")
+    with open(tflite_model_int8_path, 'wb') as f:
+      f.write(tflite_model_int8)
```
Comment (pzread) [inline common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py:101]:
```suggestion
    tflite_model_int8_path = model_dir / "model_int8.tflite"
    tflite_model_int8_path.write_bytes(tflite_model_int8)
```

**Review by pzread — APPROVED** [review]

File: common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py (hunk 2 above)
Comment (mariecwhite) [reply common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py:74]: Updated

File: common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py (hunk 1 above)
Comment (mariecwhite) [reply common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py:76]: Thanks!

File: common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py (hunk 3 above)
Comment (mariecwhite) [reply common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.py:101]: Changed

**Pull request merged**